app/assets/stylesheets/application.scss

```diff
@@ -111,7 +115,15 @@ ul .player-panel {
   margin-top: 20px;
 }
 
-ul .player-panel .user-pic :hover {
+.player-panel .user-pic :hover {
   background-image: image-url("default_prof_pic.jpg");
   opacity: 0.2;
 }
+
+main {
+  flex: 1 0 auto;
+}
+footer.page-footer {
+  background-color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.